_posts/2020-12-03-How-Create-a-New-index.html-file.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: How to Create a New index.html file
+description: 
+summary: 
+tags: 
+---
+We had trouble creating a blank index.html file, and we would forget to add the basic elements of an html page. This post is to avoid repeating the same mistake of being unsure in creating an html page. Open Up terminal and start with the initial commands up until opening Projects up.
+Once in projects, commit the following commands to create an empty index.html file:
+
+* mkdir (Name of file) - mkdir allows the user to create a folder with the name they chose to. name it
+
+* cd (File Name) - Once you have made the file, open up the file.
```

Comment:
`cd <folder_name>` -- the name you used to create the folder in the `mkdir` statement above